airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py

```diff
@@ -199,10 +199,9 @@ class Config:
         airbyte_secret=True,
     )
 
-    credentials: Optional[Union[OAuthCredentials, ServiceAccountCredentials]] = Field(
+    credentials: Union[OAuthCredentials, ServiceAccountCredentials] = Field(
         title="Authentication",
         description="Credentials for connecting to the Facebook Marketing API",
-        discriminator="auth_type",
```

Comment:
@cmm-airbyte  I'm just curious about why this used to be optional, it seems credentials are something we always wanted to be required(?).  You can look at the spec command results I left in the conversation tab. cc @bazarnov